.ado/jobs/e2e-test.yml

```diff
@@ -79,6 +79,16 @@ jobs:
                 feature: UseHermes
                 value: ${{ matrix.UseHermes }}
 
+            # NuGet ignores packages.config if it detects <PackageReference>.
+            # Use restore packages.config directly to keep compabitility with ReactNativePicker.
+            - pwsh: |
+                Get-ChildItem -Recurse -Path packages.config |`
+                  Foreach-Object {`
+                    NuGet.exe Restore -PackagesDirectory windows\packages $_`
+                  }
+              workingDirectory: packages/e2e-test-app
+              displayName: Restore packages.config items
+
```

Comment:
I thought we were going to have `run-windows` support community modules out-of-the-box, and do this there?

Reply:
Okay I see the code in the CLI to do this to, with `restorePackageConfigs()`. Is there a difference between these?

Reply:
This algorithm has been implemented into `run-windows`.
It was my understanding that this e2e-test workflow invokes MSBuild directly.

Reply:
It uses `run-windows`. See the template `run-windows-with-certificates.yml` referenced below. So I guess we're making sure it's doubly restored 🙂.

Reply:
Ah, just noticed in the runs. I must have duplicated by accident when solving restore issues similar to the ones encountered on `playground`.

I'll address this in the next PR.

Reply:
No worries. It should be fairly harmless, but good to cleanup if working in the area.